CONTRIBUTING.md

```diff
@@ -1,15 +0,0 @@
-# Contributing
```

Comment:
This file is helpful, why remove it?

Reply:
It wasn't intentional, I ended up starting the project from scratch and forgot to copy some files.